static/sidebar-single/js/grid.js

```diff
@@ -145,6 +145,40 @@
           }, '*');
         }
       };
+      var ajustElements = function(className,property,value){
+          var elements =document.getElementsByClassName(className);
+            for (var i = 0; i < elements.length; i++) {
+              switch(property){
+                  case "width":
+                    elements[i].style.width = value;
+                  break;
+                  case "display":
+                    elements[i].style.display = value;
+                  break;
+                  case "padding":
+                    elements[i].style.padding = value;
+                  break;
+                  default:
+                  //nothing
+                  break;  
+              }
+            }
+      };
+      var elementWidth = document.getElementsByClassName("tvp-sidebar-container");
+      var currentWidth = elementWidth.length>0 ? elementWidth[0].offsetWidth : 0;
+      if(currentWidth>0 && currentWidth <= 205){
+            ajustElements("tvp-video-image","width", "100%");
+            ajustElements("tvp-video-details","width", "100%");
+            ajustElements("tvp-video-views","display", "none");
+            ajustElements("tvp-video-author","display", "none");
+            ajustElements("tvp-video-details","padding", "5px 0 0 0");
+      }else{
+            ajustElements("tvp-video-image","width", null);
+            ajustElements("tvp-video-details","width", null);
+            ajustElements("tvp-video-views","display", null);
+            ajustElements("tvp-video-author","display", null);
+            ajustElements("tvp-video-details","padding", null);
+      }
       if (that.windowSize !== newSize) {
```

Comment:
Should we keep the "smart" pagination for this template? it should be always the same right? we are only changing the layout of the elements depending on small/large window size